scripts/x-charts.exports.json

```diff
@@ -88,7 +88,6 @@
   { "name": "ChartsOnAxisClickHandler", "kind": "Function" },
   { "name": "ChartsOnAxisClickHandlerProps", "kind": "Interface" },
   { "name": "ChartsPieSorting", "kind": "TypeAlias" },
-  { "name": "ChartsPluginType", "kind": "TypeAlias" },
```

Comment:
Changes in these files are a BC, but I guess it is unlikely users would be using that.

Reply:
I agree, I don't see who could have a usage of this type

You still have the possibility to mitigate the BC, but don't think it worths it

```ts
export type ChartsPluginType<T> = ChartsPlugin<T>;
```